.golangci.yml

```diff
@@ -47,6 +47,64 @@ linters:
   - whitespace
 
 linters-settings:
+  depguard:
+    rules:
+      main:
+        files:
+          - $all
+          - "!$test"
+        allow:
```

Comment:
do we have way to allow prefix like `k8s.io/` for matching all the repos in that org to keep it simple.

Reply:
tried that, it doesn't work
IMO we can drop the `depguard` linter to avoid having to maintain the config with allow, deny list (just saw CAPI dropping it from the list)

Reply:
lets do that.

Reply:
Done.